azure-pipelines.release.yml

```diff
@@ -0,0 +1,15 @@
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: recommended_practices
+      type: github
+      name: endjin/Endjin.RecommendedPractices.AzureDevopsPipelines.GitHub
+      endpoint: marain-github
+
+jobs:
+- template: templates/tag.for.release.yml@recommended_practices
+  parameters:
+    vmImage: 'windows-latest'
```

Comment:
Does this not work on Ubuntu?

Reply:
Nice to see that even with the majority of the build templated and done for us, it's still possible to introduce copy-and-paste issues...